src/components/FieldReadOnly.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactNode } from "react";
+
+interface FieldReadOnlyProps {
```

Comment:
Her vet vi allerede at vi er i kontekst av `FieldReadOnly`, så denne kan kanskje bare hete `Props`? 😊
```suggestion
interface Props {
```

Reply:
Enig.. det er tydeligvis litt både og i koden (tenkte ikke så mye over det når jeg så på koden), men hvis det er enighet rundt å bare kalle det `Props` så er det bedre .